src/adsb_actions/adsbactions.py

```diff
@@ -21,16 +20,22 @@
 from io import StringIO
 from typing import Callable
 
-from .rules import Rules
-from .flights import Flights
-from .bboxes import Bboxes
-from .stats import Stats
-from .location import Location
+from rules import Rules
+from flights import Flights
+from bboxes import Bboxes
+from stats import Stats
+from location import Location
 
 # Prometheus exporter
 from prometheus_client import start_http_server, Gauge
 
-logger = logging.getLogger(__name__)
+import adsb_logger
```

Comment:
should this be "import logging" instead, so you can shorten/clarify the references to "adsb_logger.logging" below?  I actually didn't know you could reference other modules like you have done here, so I'm not sure what the preferred style is :)

Reply:
I don't know what is pythonic for this either.  There is also hierarchy in logger with the getLogger(__name__) I'm not sure how to exploit.   I'll try it the other way.  Cleaner is nicer.